ui/storybook/package.json

```diff
@@ -7,7 +7,9 @@
     "storybook:start": "storybook dev --config-dir src/config --port 6006",
     "storybook:clean": "rimraf storybook-static/",
     "storybook:build": "npm run storybook:clean && storybook build --config-dir src/config",
-    "storybook:serve": "npx http-server ./storybook-static",
+    "storybook:serve": "npx http-server --port=6006 ./storybook-static",
+    "storybook:test": "test-storybook --config-dir src/config --stories-json",
+    "storybook:serve-and-test": "concurrently -k -s first -n \"SB,TEST\" \"npm run storybook:serve -- --silent\" \"npx wait-on tcp:6006 && npm run storybook:test\"",
```

Comment:
This is primarily used in CI where we (1) don't want to rebuild the storybook because we already did that and it can be expensive (2) we want to serve the static storybook and then run the tests against it.